src/parser/jobs/ast/modules/Divination.tsx

```diff
@@ -0,0 +1,297 @@
+				this.currentWindow.trailingGcdEvent = event
+			}
+		}
+	}
+
+	private onComplete() {
+		if (this.otherAst.countOverwritten !== 0) {
+			/*
+				SUGGESTION: Overwriting other AST's divination
+			*/
+			this.suggestions.add(new Suggestion({
+				icon: this.data.actions.DIVINATION.icon,
+				content: <Trans id="ast.divination.suggestion.usage.content">
+					Consider coordinating with your co-Astrologian to maximize the amount of time <DataLink action="DIVINATION" /> is up so the party can benefit from extra damage for longer.
+				</Trans>,
+				severity: SEVERITY.MAJOR,
+				why: <Trans id="ast.divination.suggestion.usage.why">
+					About {this.otherAst.countOverwritten} <Plural value={this.otherAst.countOverwritten} one="cast" other="casts" /> of <DataLink action="DIVINATION" /> were overwritten resulting in a loss of at least {this.parser.formatDuration(this.otherAst.timeOverwritten)}.
+				</Trans>,
+			}))
+		}
+	}
+
+	// just output, no suggestions for now.
+	override output() {
+
+		//in the case when the encounter ends prior to status remove. splicing the event to log it. note: since close event checks for null, this will not be applicable if the window isn't already open
+		const eventClose: Events['statusRemove'] = {
+			timestamp: this.parser.pull.duration,
+			status: this.data.statuses.DIVINATION.id,
+			type: 'statusRemove',
+			source: this.parser.actor.id,
+			target: this.parser.actor.id,
+		}
+		this.tryCloseWindow(eventClose)
+
+		const tableData = this.history.map(window => {
+			const end = window.end != null ?
+				window.end - this.parser.pull.timestamp :
+				this.parser.pull.duration
+			const start = window.start - this.parser.pull.timestamp
+
+			return ({
+				start,
+				end,
+				rotation: window.rotation,
+				targetsData: {
+					gcds: {
+						actual: window.gcdCount,
+						expected: (BASE_GCDS_PER_WINDOW),
+					},
+					playersBuffed: {
+						actual: window.playersBuffed.length,
+						expected: (PLAYERS_BUFFED_TARGET),
+					},
+				},
+				notesMap: {
+					otherAst: <>{this.getNotesIcon(window.overlap)}</>,
+				},
+			})
+		})
+
+		const actualCasts = tableData.length
+		const noCastsMessage = <p><span className="text-error"><Trans id="ast.divination.messages.no-casts"> There were no casts recorded for <DataLink action="DIVINATION" />.</Trans></span></p>
+
+		//for now, the message is copied directly from DivinationDowntime
+		const message = <p><Trans id="ast.ogcd-downtime.divination.description">
+			<DataLink action="DIVINATION" /> provides Astrologian with a strong amount of raid DPS when stacked together with arcanum.
+			Damage percentage bonuses stack multiplicatively, so it's most optimal to stack it with cards from <DataLink action="DRAW" /> and an Astrologian's <DataLink action="ASTRODYNE" /> when 3 unique seals are obtained. <br/>
+			Try to time the usage to match raid buffs and high output phases of other party members - it's more important to use it on time rather than hold it.
```

Comment:
Somewhat contradictory copy here, since on the one hand you're saying "use Divination when you have draw cards out and 3 Astrodyne seals" and on the other hand you're saying "use Divination with other raid buffs and party burst windows".  Maybe just remove the line about "damage percentage bonuses stack multiplicatively..."?

Reply:
Originally with the wording I was hoping it would come across as aim to use arcana, astrodyne, and divination along with party burst windows, but I see that it did not come across as such
Removed line as suggested for clarity